proto/ibc/core/connection/v1/tx.proto

```diff
@@ -12,34 +12,31 @@ import "ibc/core/connection/v1/connection.proto";
 // Msg defines the ibc/connection Msg service.
 service Msg {
   // ConnectionOpenInit defines a rpc handler method for MsgConnectionOpenInit.
-  rpc ConnectionOpenInit(MsgConnectionOpenInit)
-      returns (MsgConnectionOpenInitResponse);
+  rpc ConnectionOpenInit(MsgConnectionOpenInit) returns (MsgConnectionOpenInitResponse);
 
   // ConnectionOpenTry defines a rpc handler method for MsgConnectionOpenTry.
-  rpc ConnectionOpenTry(MsgConnectionOpenTry)
-      returns (MsgConnectionOpenTryResponse);
+  rpc ConnectionOpenTry(MsgConnectionOpenTry) returns (MsgConnectionOpenTryResponse);
 
   // ConnectionOpenAck defines a rpc handler method for MsgConnectionOpenAck.
-  rpc ConnectionOpenAck(MsgConnectionOpenAck)
-      returns (MsgConnectionOpenAckResponse);
+  rpc ConnectionOpenAck(MsgConnectionOpenAck) returns (MsgConnectionOpenAckResponse);
 
   // ConnectionOpenConfirm defines a rpc handler method for
   // MsgConnectionOpenConfirm.
-  rpc ConnectionOpenConfirm(MsgConnectionOpenConfirm)
-      returns (MsgConnectionOpenConfirmResponse);
+  rpc ConnectionOpenConfirm(MsgConnectionOpenConfirm) returns (MsgConnectionOpenConfirmResponse);
 }
 
 // MsgConnectionOpenInit defines the msg sent by an account on Chain A to
 // initialize a connection with Chain B.
 message MsgConnectionOpenInit {
-  option (gogoproto.equal) = false;
+  option (gogoproto.equal)           = false;
   option (gogoproto.goproto_getters) = false;
 
-  string client_id = 1 [ (gogoproto.moretags) = "yaml:\"client_id\"" ];
-  Counterparty counterparty = 2 [ (gogoproto.nullable) = false ];
-  Version version = 3;
-  uint64 delay_period = 4 [ (gogoproto.moretags) = "yaml:\"delay_period\"" ];
-  string signer = 5;
+  string       client_id           = 1 [(gogoproto.moretags) = "yaml:\"client_id\""];
+  Counterparty counterparty        = 2 [(gogoproto.nullable) = false];
+  Version      version             = 3;
+  uint64       delay_time_period   = 4 [(gogoproto.moretags) = "yaml:\"delay_time_period\""];
+  uint64       delay_height_period = 5 [(gogoproto.moretags) = "yaml:\"delay_height_period\""];
+  string       signer              = 6;
```

Comment:
modifying proto field numbers is protobuf breaking. That means we should either create a v2 or follow the [protobuf update recommendations](https://developers.google.com/protocol-buffers/docs/overview#updating)

You should just be able to append the delay height

Reply:
this would allow relayers to use the new protobuf while still successfully submitting messages to chains using the old code